department-of-veterans-affairs/abd-vro
# MCP-2765 Hypertension Template Spacing

## What was the problem?

https://amida.slack.com/archives/C01Q7979Z7D/p1681765155628689

30px gap was missing not being properly applied to one of the sections

Requested:
![image](https://user-images.githubusercontent.com/13013464/232806022-6f4def1a-6fa5-4e9a-9e3f-8dc7e55bcf3b.png)

Current:
![image](https://user-images.githubusercontent.com/13013464/232806051-a65406e1-3cb5-4fa2-8430-e9c1c67d7cf6.png)


Associated tickets or Slack threads:

- [MCP-2765](https://amida.atlassian.net/browse/MCP-2765)

## How does this fix it?[^1]

Moves the 30px class to the main parent container

## How to test this PR
- Generate a PDF and compare to the images


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
